src/main/java/org/broadinstitute/consent/http/db/DataAccessRequestDAO.java

```diff
@@ -32,7 +32,8 @@ public interface DataAccessRequestDAO extends Transactional<DataAccessRequestDAO
   @SqlQuery(
       "SELECT id, reference_id, collection_id, parent_id, draft, user_id, create_date, sort_date, submission_date, update_date, (data #>> '{}')::jsonb AS data FROM data_access_request "
           + "  WHERE not (data #>> '{}')::jsonb ??| array['partial_dar_code', 'partialDarCode'] "
-          + "  AND draft != true ")
+          + "  AND draft != true "
+          + "  AND (data->>'status' != 'Archived' OR data->>'status' IS NULL ) ")
```

Comment:
Same thing in this file
```suggestion
          + "  AND (LOWER(data->>'status') != 'archived' OR data->>'status' IS NULL ) ")
```